doc/Developing.md

```diff
@@ -138,6 +138,53 @@ When a PR is merged into `develop`, SNAPSHOT packages are automatically publishe
 
 ## Releasing
 
+### Sonatype Nexus publishing setup
+
+- Have `dnanexus`'s password for [Sonatype Nexus repository manager](https://oss.sonatype.org/#stagingRepositories).
```

Comment:
Added where to get the password from. Not sure about the reasons to publish there or feasibility of doing it through GHA, will set up a discussion w/ @commandlinegirl  and @jdidion .

Reply:
We publish there so that code that depends on the packages can be built without access to our GitHub packages. It might be possible to do it with GHA but I have tried and not been successful.